glean_parser/schemas/metrics.1-0-0.schema.yaml

```diff
@@ -111,8 +130,8 @@ definitions:
         title: Send in pings
         description: |
           Which pings the metric should be sent on. If not specified, the metric
-          is sent on the "default ping", which is the "events" ping for events,
-          and the "metrics" ping for everything else. Most metrics don't need to
+          is sent on the "default ping", which is the `events` ping for events,
```

Comment:
nit: let's also mention the `default` keyword here, used to represent default pings